ssc/Pos/Ssc/Base.hs

```diff
@@ -78,27 +73,6 @@ secretToSharedSeed = SharedSeed . getDhSecret . secretToDhSecret
 vssThreshold :: Integral a => a -> Threshold
 vssThreshold len = fromIntegral $ len `div` 2 + len `mod` 2
 
--- | Generate random SharedSeed.
-genCommitmentAndOpening
```

Comment:
Moved to https://github.com/input-output-hk/cardano-sl/pull/3089/files#diff-c617615e5acedeeb64d82fc1ab230952R19 (and renamed)